mlflow/utils/file_utils.py

```diff
@@ -967,3 +969,26 @@ def chdir(path: str) -> None:
         yield
     finally:
         os.chdir(cwd)
+
+
+def get_total_file_size(path: str) -> int:
```

Comment:
Can we use https://docs.python.org/3/library/shutil.html#shutil.disk_usage?

Reply:
I don't think `shutil.disk_usage` returns what we want. Given a folder that contain roughly ~99MB files, it returns `usage(total=1000240963584, used=902191882240, free=98049081344)`, which seems to be way off. Actually I don't quite understand what it intends to return here.

Reply:
You're right! I misunderstood what `shutil.disk_usage` does :)